configuration.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python3
+
+###########################################################################
+#
+#    Copyright 2024 Google LLC
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#            https://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+###########################################################################
+
+"""Module that defines global parameters"""
+
+import os
+
+FFMPEG_BUFFER = "reduced/buffer.mp4"
```

Comment:
Are these intended to be public?

Reply:
Yes, we left them out of the class because they are typically configured at the application layer not at instantiation.